packages/wol.rb

```diff
@@ -3,28 +3,20 @@
 class Wol < Package
   description 'Wake up hardware that is Magic Packet compliant'
   homepage 'http://ahh.sourceforge.net/wol/'
-  version '0.7.1-r1'
+  version '0.7.1'
   source_url 'https://downloads.sourceforge.net/ahh/wol-0.7.1.tar.gz'
   source_sha256 'e0086c9b9811df2bdf763ec9016dfb1bcb7dba9fa6d7858725b0929069a12622'
 
   depends_on 'compressdoc' => :build
 
   def self.build
-     system"./configure \
-            --prefix=#{CREW_PREFIX} \
-            --enable-gui=no \
-            --with-features=huge \
-            --without-x \
-            --disable-nls \
-            --enable-multibyte \
-            --with-tlib=ncurses \
-            --enable-perlinterp \
-            --disable-selinux"
+    system "./configure", "--prefix=#{CREW_PREFIX}", "--mandir #{CREW_PREFIX}/share/man"
```

Comment:
Missing `=` in between `--mandir` and `#{CREW_PREFIX}/share/man`?

Reply:
Whoops, guess that's what happens when you make last-minute cosmetic changes without checking. I've fixed this and will do one last test to make sure everything functions properly.